packages/altive_lints/lib/altive_lints.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+
+import 'src/lints/avoid_hardcoded_japanese.dart';
+
+PluginBase createPlugin() => _AltivePlugin();
+
+class _AltivePlugin extends PluginBase {
+  @override
+  List<LintRule> getLintRules(CustomLintConfigs configs) => [
+        const AvoidHardcodedJapanese(),
```

Comment:
As an example, I have defined custom lint rules that have already been implemented in another repository.